torch/cuda/__init__.py

```diff
@@ -377,7 +430,17 @@ def get_gencode_flags() -> str:
     arch_list_ = [arch.split("_") for arch in arch_list]
     return " ".join([f"-gencode compute=compute_{arch},code={kind}_{arch}" for (kind, arch) in arch_list_])
 
+def set_stream(stream: Stream):
```

Comment:
This is not meant to be called by users so I think we should hide it in `_utils` or `_internal` or something.